modules/uhttp/http.go

```diff
@@ -78,7 +78,7 @@ type Config struct {
 	modules.ModuleConfig
 	Port    int           `yaml:"port"`
 	Timeout time.Duration `yaml:"timeout"`
-	Debug   *bool         `yaml:"debug"`
+	Debug   bool          `yaml:"debug" default:"true"`
```

Comment:
Actually, I was wrong: it will work on all providers, because we iterate over a struct to get values and take defaults from the object itself.